graph/resolvers/category.go

```diff
@@ -68,6 +68,11 @@ func (r *queryResolver) Category(ctx context.Context, id int) (*models.Category,
 }
 
 func (r *queryResolver) Categories(ctx context.Context, ids []int, spaces []int, page *int, limit *int, sortBy *string, sortOrder *string) (*models.CategoriesPaging, error) {
```

Comment:
Should remove `spaces []int` param from schema.graphql, right?
Should remove from every query resolver.

Reply:
Right now we will not use it but in future we need it

Reply:
👍🏼